src/index.js

```diff
@@ -9,6 +9,10 @@ import './index.css'
 // Reminders
 import { initReminders } from './reminder'
 
+// Service Worker
```

Comment:
Why did you move the registration into index.js?

Reply:
Apps generated using `create-react-app` has `serviceWorker.js` register in `index.js` itself. Also, it wouldn't make much difference in terms of performance as loading it in `DynamicApp.js` or `index.js` is one and the same and the script won't execute anyway until the `window` is successfully loaded.